File: petya.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	string one, two;
	cin >> one >> two;
	if (lexicographical_compare(one.begin(), one.end(), two.begin(), two.end())) {
		cout << -1 << "\n";
	} else if (one == two) {
		cout << 0 << "\n";
	} else {
		cout << 1 << "\n";
	}
}

File: petya.java
import java.util.*;

class petya {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		String one = sc.nextLine();
		String two = sc.nextLine();
		if (one.compareTo(two) > 0) {
			System.out.printn(1);
		} else if (one.compareTo(two) < 0) {
			System.out.println(-1);
		} else {
			System.out.println(0);
		}
	}
}